samples/Samples.StartupHook.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -0,0 +1,49 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:59775",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+
+        "DOTNET_STARTUP_HOOKS": "$(StartupHookOutputDirectory)\\netcoreapp3.1\\OpenTelemetry.Instrumentation.StartupHook.dll",
+        "OTEL_MANAGED_PROFILER_PATH": "$(ManagedProfilerOutputDirectory)\\netcoreapp3.1",
```

Comment:
This part is a bit conflicting with the nature of auto instrumentation.  Currently we do not specify managed framework, it's picked up automatically by the loader. Requiring this would need app local settings always to be present, instead of using global one.

Reply:
I agree. Called this out in the PR description. Will use `OTEL_DOTNET_TRACER_HOME`.